app/src/full/java/io/homeassistant/companion/android/settings/views/SettingsWearHomeView.kt

```diff
@@ -0,0 +1,42 @@
+package io.homeassistant.companion.android.settings.views
+
+import android.app.Activity
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalContext
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.google.android.material.composethemeadapter.MdcTheme
+import io.homeassistant.companion.android.settings.SettingsWearViewModel
+
+@Composable
+fun LoadSettingsHomeView(
+    settingsWearViewModel: SettingsWearViewModel,
+    deviceName: String,
+    activity: Activity
+) {
+    val context = LocalContext.current
+    MdcTheme {
+        val navController = rememberNavController()
+        NavHost(navController = navController, startDestination = SettingsWearMainView.LANDING) {
+            composable(SettingsWearMainView.FAVORITES) {
+                LoadWearFavoritesSettings(
+                    settingsWearViewModel.entities,
+                    settingsWearViewModel.favoriteEntityIds.toList(),
+                    { b: Boolean, s: String, a: Activity -> settingsWearViewModel.onEntitySelected(b, s, a) },
+                    {
+                        settingsWearViewModel.favoriteEntityIds.toList().contains(
+                            settingsWearViewModel.favoriteEntityIds.toList()[it]

```

Comment:
I noticed a bug where trying to deselect an entity in some cases results in duplicating the entity (both still selected). I think it is due to lines 28 and 29. This should return whether the entity is selected.

Reply:
So when an entity is selected it actually moves to the favorite list, then once deselected from the favorite it goes back down to the entity list. Does it actually pass that to the device?

Reply:
So I'm not sure if I understand your question. But in the log I see that the flow for sending the changes to the wear device is followed correctly.

In any case, I'm also not sure if I'm pointing to the correct source of the bug. But I don't understand how these two lines are supposed to work. Loosely translated it states: `list.contains(list[index])`. I'm not sure how that resolves to whether the  entity is selected as a favorite.

Let me also better describe the bug. I'm trying to deselect one of the favorite entities, which are displayed on the top with selected items. If I press it, instead of deselecting it duplicates the item. So the same entity is shown twice at the top of the list, both selected. If I press again, it duplicates again.
When I leave the screen and come back again, the list is normal again.
This does not always happen. Sometimes I can deselect an entity without problems.

Reply:
ok let me try and play with it while I look into getting a receiver setup to update the favorite view model.  The purpose of the call is to determine if the favorite item should be checked or not, its similar to what we do set favorites. I think because it doesn't always happen might suggest a timing issue where the list is updated slower than when the user selects the entity maybe?